desktop/extensions-sdk/extensions/publish.md

```diff
@@ -6,21 +6,24 @@ keywords: Docker, extensions, publish
 
 ## Submit your extension to the marketplace
 
-Docker Desktop displays published extensions in [the Extensions Marketplace](https://hub.docker.com/search?q=&type=extension){: target="_blank" rel="noopener" class="_" }. The Extensions Marketplace is a curated space where developers can discover extensions to improve their developer experience and propose their own extension to be available for all Desktop users.
+Docker Desktop displays published extensions in the Extensions Marketplace on [Docker Desktop](https://open.docker.com/extensions/marketplace){: target="_blank" rel="noopener" class="_"} and [Docker Hub](https://hub.docker.com/search?q=&type=extension){: target="_blank" rel="noopener" class="_"}.
+The Extensions Marketplace is a curated space where developers can discover extensions to improve their developer experience and propose their own extension to be available for all Desktop users.
 
-If you want to publish your extension in the Marketplace, you can submit your extension [here](https://www.docker.com/products/extensions/submissions/){: target="_blank" rel="noopener" class="_" }.
+Whenever you feel ready to publish your extension in the Marketplace, you can submit it by creating an issue on [this repository](https://github.com/docker/extensions-submissions){: target="_blank" rel="noopener" class="_" }.
 
-All extensions submitted to the Extension Marketplace are reviewed and approved by the Docker Extension team before listing. This review process ensures a level of trust, security, and quality for developers using Docker Extensions and allows for extension developers to get feedback.
+From there you can choose either to request a review from the Docker Extension team before listing, or not. This review process ensures a level of trust, and quality for developers using Docker Extensions and allows for extension developers to get feedback.
+However, it makes the publishing process longer. If you don't want to request a review, some automatic checks will be performed before listing your extension. 
 
 ### Before you submit
 
-Ensure your extension has followed the guidelines outlined in this section before submitting for your extension for review. Docker highly encourages you to check the guidelines as not doing so may considerably impact the duration of the approval process.
+Before you submit your extension, it must be [validated](validate.md).
+
+Ensure your extension has followed the guidelines outlined in this section before submitting for your extension for review. Docker highly encourages you to check the guidelines to increase the user adoption of your extension. In case you requested a review from the Docker Extension team, not doing so may considerably impact the duration of the process.
```

Comment:
```suggestion
It is highly recommended your extension follows the guidelines outlined in this section before submitting your extension. In case you requested a review from the Docker Extension team, not doing so may considerably impact the duration of the process.
```